packages/test/src/integration-tests.ts

```diff
@@ -918,6 +918,38 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     await handle.terminate();
   });
 
+  test('throwBigInt includes message with our recommendation', async (t) => {
+    const { client } = t.context;
+    const workflowId = uuid4();
+    const handle = await client.start(workflows.throwBigInt, {
+      taskQueue: 'test',
+      workflowId,
+    });
+    await asyncRetry(
+      async () => {
+        const history = await client.service.getWorkflowExecutionHistory({
+          namespace: 'default',
+          execution: { workflowId },
+        });
+        const wftFailedEvent = history.history?.events?.find((ev) => ev.workflowTaskFailedEventAttributes);
+        if (wftFailedEvent === undefined) {
+          throw new Error('No WFT failed event');
+        }
+        const failure = wftFailedEvent.workflowTaskFailedEventAttributes?.failure;
+        if (!failure) {
+          t.fail();
+          return;
+        }
+        t.is(
+          failure.message,
+          '42 [A non-Error value was thrown from your code. We recommend throwing Error objects so that we can provide a stack trace.]'
```

Comment:
Let's remove the period at the end of this.
In general, our error message should include periods IMHO.